src/REPLMode.jl

```diff
@@ -901,7 +893,7 @@ command_declarations = [
     CMD_REGISTRY_ADD,
     ["add"],
     do_registry_add!,
-    (ARG_PKG, []),
+    ([], nothing, []),
```

Comment:
We can also use `identity` as a parser and change `Union{Function,Nothing}` with judt `Function`.